lib/iris/_data_manager.py

```diff
@@ -37,7 +37,7 @@ class DataManager(object):
 
     """
 
-    def __init__(self, data, fill_value=None, realised_dtype=None):
+    def __init__(self, data, fill_value='none', realised_dtype=None):
```

Comment:
Having `fill_value='none'` is akin to the usage in `DataManager.copy`, in that it allows for a masked array to be passed to the constructor alone, and the masked array `fill_value` will propagate to the `DataManager.fill_value`. To override this behaviour for a masked array is achieved by explicitly setting the `fill_value` i.e. `DataManager(masked_array, fill_value=None)`